test/instrumentation/rest-client_test.rb

```diff
@@ -221,7 +221,7 @@
     traces[13]['ServiceArg'].must_equal '/'
     traces[13]['HTTPMethod'].must_equal 'GET'
     traces[13]['HTTPStatus'].must_equal "200"
-    traces[13].key?('Backtrace').must_equal AppOpticsAPM::Config[:nethttp][:collect_backtraces]
+    traces[13].key?('Backtrace').must_equal !!AppOpticsAPM::Config[:rest_client][:collect_backtraces]
```

Comment:
Note sure why these changed - assuming it's OK.

Reply:
good catch! 
I had to add the `!!` and assumed that rest-client tests would test rest_client stuff ;)